jobs/check-assessment-exercise-ids.groovy

```diff
@@ -0,0 +1,41 @@
+// Loads all the assessment configs from datastore and checks to make sure that
+// their ExerciseIDs are valid (i.e., that there are published exercises with
+// those IDs).
+
+@Library("kautils")
+// Classes we use, under jenkins-jobs/src/.
+import org.khanacademy.Setup;
+
+
+new Setup(steps).apply();
+
+
+def _setupWebapp() {
+   withTimeout('1h') {
+      kaGit.safeSyncToOrigin("git@github.com:Khan/webapp", "master");
+   }
+}
+
+
+def runScript() {
+   withTimeout('1h') {
+      dir("webapp") {
+         sh("go run services/assessments/cmd/check_assessments_exercise_ids/main.go");
```

Comment:
This is fine, and certainly very common in our jenkins code, but I think it's a bit safer to do exec:
```
    exec(["go", "run", "services/..."]);
```
It's totally up to you.